mozregression/utils.py

```diff
@@ -60,6 +61,15 @@ def names(self):
 CACHE_SESSION = None
 
 
+def retry_get(**kwargs):
```

Comment:
Maybe a better way to facilitate calls is to mimic the get behaviour only. (Sorry, I cahnged my maind about it :()
Something like the following will simplify the calls:
def retry_get(url, **kwargs):
    return redo.retry(url, kwargs=kwargs, ...)